1906-10-05.xml

```diff
@@ -0,0 +1,7096 @@
+                        primarily one for the Moslem people of Turkey, and for a foreign Govern.
+                        ment to pronounce an opinion one way or another would have the appearance of
+                        mixing in a Palace intrigue. If it be said that Turkey is more or less under
+                        European tutelage, and that the unanimous consent of Europe to s change of
+                        Moslem law would give it a sanction which would secure its general
+                        acceptance, my reply would be that unanimity on this subject is impossible,
+                        and to attempt to obtain it would occasion endless intrigue. It may be
+                        admitted that if the general consent of Europe could be obtained it might
+                        establish an important precedent. But if such consent has been applied for,
+                        and il, as is rumoured, negotiations are going on, it is because opposition
+                        is anticipated either by Turkey or by some foreign State. Even an attempt by
+                        the Concert of Europe would be denounced by the Moslem population as an
+                        interference by the giaours with their religious rights, | There is much to
+                        be said for and much against the alleged project. The generally expressed
+                        opinion is that certain Palace favourites, for their own purposes, are in
+                        its favour, and this would not recommend it to the Turkish population, and
+                        still less to Europeans. Indeed, in whatever way the expe. diency of a
+                        change in the law of succession may be regarded, its practical application
+                        will be dangerous. It will almost certainly lead to disturbances, and it
+                        ought to be a question for Turkish statesmen whether it is desirable to add
+                        to the many difficulties with which they have to deal a possible war of
+                        succession. For foreign States the question is at once tempting and
+                        dangerous. In the solution of the many questions in the Near East with which
+                        our country is concerned, it is difficult to see what advantage could be
+                        gained by entering upon it. Sir Edward Grey will probably conclude that his
+                        wisest course is to reserve speech or action, and take up the poste tion
+                        that Great Britain will stand aloof, and neither approve nor disapprove any
+                        action of the kind which the Palace advisers may choose to take.</p>
+                    <p><!--"The Sultan's Successor" was split into two different sections by an ad, so I just marked it as "Cont."--></p>
```

Comment:
thanks for the comment and for noticing this problem. I still have not figured out an efficient way to deal with this structure. The only thing I'll say is don't add the extra title--the only text that should appear (outside of comments) is the text on the original page.